src/components/DatePicker/DatePicker.test.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { fireEvent, render, screen, waitFor } from '../../test/testUtils';
+import { DatePicker, DatePickerProps } from './DatePicker';
+
+const defaultProps: DatePickerProps = {
+  yearRange: { start: 1990, end: 2100 },
+  locale: 'de',
+};
+
+const setup = (props = {}) => {
+  const combinedProps = { ...defaultProps, ...props };
+  render(<DatePicker {...combinedProps} />);
+};
+
+describe('The DatePicker component', () => {
+  it('displays a field with the current date ', () => {
+    setup();
+
+    expect(screen.getByTestId('datepicker-input1')).toHaveValue(
+      `${new Date().toLocaleDateString('de')}`
+    );
+  });
+
+  it('changes the date value in the field if the user clicks on another date in the calender', async () => {
+    setup();
+
+    fireEvent.click(screen.getByTestId('datepicker-input1'));
+    await waitFor(() =>
+      fireEvent.change(screen.getByTestId('datepicker-input1'), { target: { value: '10.10.2022' } })
```

Comment:
You could use a different number for month and day to ensure that the order does not get mixed up
```suggestion
      fireEvent.change(screen.getByTestId('datepicker-input1'), { target: { value: '20.04.2022' } })
```